fiftyone/operators/delegated.py

```
@@ -811,11 +812,30 @@ async def _execute_operator(self, doc):
+                if parent_doc.pipeline and parent_doc.pipeline_run_info:
+                    stage_index = parent_doc.pipeline_run_info.stage_index
+                    pipeline_ctx = PipelineExecutionContext(
+                        parent_doc.pipeline_run_info.active,
+                        stage_index,
+                        len(parent_doc.pipeline.stages),
+                        num_distributed_tasks=parent_doc.pipeline.stages[
+                            stage_index
+                        ].get("num_distributed_tasks")
+                        or 0,
+                    )
+        except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)